src/bit_cuda/bit_cuda_transformer.h

```diff
@@ -133,6 +147,15 @@ class BitCudaTransformer : public CfgVisitor<LatticeType>
             }
         } 
     }
+    void add_id(int bitcudaId, int transfer[]){
```

Comment:
Maybe `add_taint_sources` is a more descriptive name

Reply:
and `bitcudaId` -> `nodeIndex`

Reply:
renamed them